src/services/streetView.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// @ts-ignore
+import asyncLoading from 'react-async-loader';
```

Comment:
Even if there is no types file for this package, Why `@ts-ignore` is required?
Is it referring to this issue?
https://github.com/microsoft/typescript/issues/3019